joomlatools/joomlatools-framework
# Wrong CSS font paths on Joomla 4

fixes #593 

### Test Instructions

Run a Joomla 4 site from a folder and check that all of the editors icons are properly rendered and that the inspector doesn't report any bad font links.
